packages/web/src/shared/menu-items.ts

```diff
@@ -15,6 +15,10 @@ export const menuItems = {
     name: 'Build',
     link: '/build',
   },
+  VALIDATORS: {
```

Comment:
would you mind calling this one `VALIDATORS_LIST` there are plans for a top level validators page